pkg/tasks/grafana.go

```diff
@@ -14,12 +14,10 @@
 
 package tasks
 
+// TODO this file can be be removed in 4.12
 import (
 	"context"
 
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	"github.com/openshift/cluster-monitoring-operator/pkg/client"
 	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
 	"github.com/pkg/errors"
```

Comment:
Unused package local var grafanaLegacyDataSourceSecretName should be removed too.

Reply:
Done